test/unittests/test_excel_to_json/test_properties.py

```diff
@@ -0,0 +1,592 @@
+"""unit tests for excel to properties"""
+
+# pylint: disable=missing-class-docstring,missing-function-docstring,disable=protected-access,
+# disable=wrong-import-order mypy: allow_untyped_calls
```

Comment:
```suggestion
# pylint: disable=missing-class-docstring,missing-function-docstring,protected-access,wrong-import-order
# mypy: allow_untyped_calls
```

Does `mypy: allow_untyped_calls` have an effect? If I remove it and run mypy, nothing happens...

Reply:
I'll try it out

Reply:
Works for me too